src/utils/map-state.js

```diff
@@ -6,10 +6,7 @@ export const MAPBOX_LIMITS = {
   minZoom: 0,
   maxZoom: 20,
   minPitch: 0,
-  maxPitch: 60,
-  // defined by mapbox-gl
-  maxLatitude: 85.05113,
```

Comment:
FWIW - Not sure where these ended up, if at all. But they are defined by the web mercator projection, not mapbox, it is the cutoff at the lat that makes the projected map square.

Reply:
This is moved to `viewport-mercator-project`'s `normalizeViewportProps` util.